frontend/src/containers/data-tool/content/details/mocked-data.ts

```diff
@@ -0,0 +1,67 @@
+import { DashboardTableItem } from '@/components/dashboard-table';
+
+export const mockedData: DashboardTableItem[] = [
```

Comment:
Do we still need mocked data declared on our end? If so, I would create the data in the CMS and pull it as it was real, so we have the hooks and implementation ready and it is a matter of updating the CMS with real data.

Reply:
The tables are being done in subsequent tasks.

I've only only added them because we already had these (and the mocked data; it was just copied) already available in the code, so that:  
- The details pane is not empty  
- To test the layout overflows  
The table/data in it doesn't reflect what will be implemented. 

The next task in the sprint (according to the sprint planning breakdown) is [SKY30-88](https://vizzuality.atlassian.net/browse/SKY30-88), which although still uses mocked data, it'll change all this. The code for that particular task is nearly finished, and is based on this PR (because it introduces different tables depending on the location type).

After that we have tasks to start pulling the data from the API and actually populate the tables; I believe by now the endpoints are already available or about to be. :)